### PR TITLE
Pangram: check case insensitive behavior

### PR DESCRIPTION
**Pull request by iHiD**
This is a re-opening of https://github.com/exercism/problem-specifications/pull/1658

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
I've rebased this on the latest main and made one change, inspired by https://github.com/exercism/problem-specifications/pull/1658#issuecomment-633702171: removing the second test case.

CC @exercism/reviewers

**Review by petertseng — APPROVED** [review]
I'm not huge on the necessity of this case, because supposedly case 2577bf54-83c8-402d-a64b-a2c0f7bb213a "case insensitive" already handles it.

```
$ ruby -e 'p ("the quick brown fox jumps over with lazy FX".chars - [" "]).uniq.size' 
26
```

But, this is certainly a simpler and easier-to-understand way of expressing the same thing. Maybe it could even go before case 2577bf54-83c8-402d-a64b-a2c0f7bb213a "case insensitive" ...

File: exercises/pangram/canonical-data.json [hunk 1]
```diff
@@ -94,6 +94,15 @@
         "sentence": "the quick brown fox jumps over with lazy FX"
       },
       "expected": false
+    },
+    {
+      "uuid": "7138e389-83e4-4c6e-8413-1e40a0076951",
+      "description": "a-m and A-M are 26 different letters but not a pangram",
```
Comment (ee7) [inline exercises/pangram/canonical-data.json:100]:
```suggestion
      "uuid": "7138e389-83e4-4c6e-8413-1e40a0076951",
      "reimplements": "2577bf54-83c8-402d-a64b-a2c0f7bb213a",
      "description": "a-m and A-M are 26 different letters but not a pangram",
```

Would this make sense?

The sentence for that test case is pretty weird:
> "the quick brown fox jumps over with lazy FX"

and makes the reader wonder about "with" and "FX".

So if we think they test the same thing, we could just replace it.

Comment (ErikSchierboom) [reply exercises/pangram/canonical-data.json:100]: I like that.

**Review by petertseng — APPROVED** [review]
I think they test the same thing. Replacing seems a fine idea.

**Review by SleeplessByte — APPROVED** [review]

File: exercises/pangram/canonical-data.json [hunk 2]
```diff
@@ -94,6 +94,16 @@
         "sentence": "the quick brown fox jumps over with lazy FX"
       },
       "expected": false
+    },
+    {
+      "uuid": "7138e389-83e4-4c6e-8413-1e40a0076951",
+      "reimplements": "2577bf54-83c8-402d-a64b-a2c0f7bb213a",
+      "description": "a-m and A-M are 26 different letters but not a pangram",
```
Comment (ee7) [inline exercises/pangram/canonical-data.json:101]:
```suggestion
      "description": "a-m and A-M are 26 different characters but not a pangram",
```

Let's avoid "letters" here? 'a' and 'A' are the same letter.

**Review by ee7 — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Thanks all!

**Pull request merged**